packages/theme/brand/altinn.css

```diff
@@ -637,61 +637,52 @@
-  --ds-border-radius-3: min( var(--ds-border-radius-base) , var(--ds-border-radius-scale) *5);
-  --ds-border-radius-4: var(--ds-border-radius-base);
-  --ds-border-radius-5: 624.9375rem;
-  --ds-border-radius-base: 0.25rem;
-  --ds-border-radius-scale: 0.25rem;
-  --ds-border-radius-sm: var(--ds-border-radius-1);
-  --ds-border-radius-md: var(--ds-border-radius-2);
-  --ds-border-radius-lg: var(--ds-border-radius-3);
-  --ds-border-radius-default: var(--ds-border-radius-4);
-  --ds-border-radius-full: var(--ds-border-radius-5);
+  --ds-border-radius-sm: min( 0.25rem , 0.25rem *1);
+  --ds-border-radius-md: min( 0.25rem , 0.25rem *2);
+  --ds-border-radius-lg: min( 0.25rem , 0.25rem *5);
+  --ds-border-radius-default: 0.25rem;
+  --ds-border-radius-full: 624.9375rem;
   --ds-disabled-opacity: 30%;
-  --ds-spacing-0: calc(var(--ds-spacing-scale)*0);
-  --ds-spacing-1: calc(var(--ds-spacing-scale)*1);
-  --ds-spacing-2: calc(var(--ds-spacing-scale)*2);
-  --ds-spacing-3: calc(var(--ds-spacing-scale)*3);
-  --ds-spacing-4: calc(var(--ds-spacing-scale)*4);
-  --ds-spacing-5: calc(var(--ds-spacing-scale)*5);
-  --ds-spacing-6: calc(var(--ds-spacing-scale)*6);
-  --ds-spacing-7: calc(var(--ds-spacing-scale)*7);
-  --ds-spacing-8: calc(var(--ds-spacing-scale)*8);
-  --ds-spacing-9: calc(var(--ds-spacing-scale)*9);
-  --ds-spacing-10: calc(var(--ds-spacing-scale)*10);
-  --ds-spacing-11: calc(var(--ds-spacing-scale)*11);
-  --ds-spacing-12: calc(var(--ds-spacing-scale)*12);
-  --ds-spacing-13: calc(var(--ds-spacing-scale)*13);
-  --ds-spacing-14: calc(var(--ds-spacing-scale)*14);
-  --ds-spacing-15: calc(var(--ds-spacing-scale)*15);
-  --ds-spacing-18: calc(var(--ds-spacing-scale)*18);
-  --ds-spacing-22: calc(var(--ds-spacing-scale)*22);
-  --ds-spacing-26: calc(var(--ds-spacing-scale)*26);
-  --ds-spacing-30: calc(var(--ds-spacing-scale)*30);
-  --ds-spacing-scale: 0.25rem;
-  --ds-sizing-0: calc(var(--ds-sizing-scale)*0);
-  --ds-sizing-1: calc(var(--ds-sizing-scale)*1);
-  --ds-sizing-2: calc(var(--ds-sizing-scale)*2);
-  --ds-sizing-3: calc(var(--ds-sizing-scale)*3);
-  --ds-sizing-4: calc(var(--ds-sizing-scale)*4);
-  --ds-sizing-5: calc(var(--ds-sizing-scale)*5);
-  --ds-sizing-6: calc(var(--ds-sizing-scale)*6);
-  --ds-sizing-7: calc(var(--ds-sizing-scale)*7);
-  --ds-sizing-8: calc(var(--ds-sizing-scale)*8);
-  --ds-sizing-9: calc(var(--ds-sizing-scale)*9);
-  --ds-sizing-10: calc(var(--ds-sizing-scale)*10);
-  --ds-sizing-11: calc(var(--ds-sizing-scale)*11);
-  --ds-sizing-12: calc(var(--ds-sizing-scale)*12);
-  --ds-sizing-13: calc(var(--ds-sizing-scale)*13);
-  --ds-sizing-14: calc(var(--ds-sizing-scale)*14);
-  --ds-sizing-15: calc(var(--ds-sizing-scale)*15);
-  --ds-sizing-18: calc(var(--ds-sizing-scale)*18);
-  --ds-sizing-22: calc(var(--ds-sizing-scale)*22);
-  --ds-sizing-26: calc(var(--ds-sizing-scale)*26);
-  --ds-sizing-30: calc(var(--ds-sizing-scale)*30);
-  --ds-sizing-scale: 0.25rem;
+  --ds-spacing-0: 0rem;
```

Comment:
Need output reference for spacing & sizing still.

Reply:
Yes, I was wondering about this. I'll look into it – is the reason just so that it is possible to e.g. override `--ds-sizing-scale` for a single component?